x-pack/metricbeat/module/cockroachdb/_meta/fields.yml

```diff
@@ -1,10 +1,21 @@
 - key: cockroachdb
   title: 'CockroachDB'
-  release: beta
+  release: ga
   description: >
-    CockroachDB module
+    CockroachDB module scrape metrics using Prometheus endpoint.
   settings: ["ssl", "http"]
   fields:
-    - name: cockroachdb
+    - name: prometheus
       type: group
       fields:
+        # Order is important here, labels will match first, the rest are double
+        - name: labels.*
+          type: object
+          object_type: keyword
+          description: >
+            Prometheus metric labels
+        - name: metrics.*
+          type: object
+          object_type: double
+          object_type_mapping_type: "*"
+          description: >
```

Comment:
These fields are already added by the prometheus module. As Metricbeat works, there is a single index template, and each field can be defined only once.

This is what causes the problems in CI:
```
Exception: export command returned with an error: Error generating Index Pattern: field <prometheus.labels.*> is duplicated, remove it or set 'overwrite: true', {Name:labels.* Type:object Description:Prometheus metric labels
```